directory.td.json

```diff
@@ -16,9 +16,38 @@
                 "read",
                 "search"
             ]
+        },
+        "oauth2_client": {
+            "scheme": "oauth2",
+            "flow": "client",
+            "token": "https://auth.example.com/token",
+            "scopes": [
+                "write",
+                "read",
+                "search"
+            ]
+        },
+        "oauth2_device": {
+            "scheme": "oauth2",
+            "flow": "device",
+            "authorization": "https://auth.example.com/device_authorization",
+            "token": "https://auth.example.com/token",
+            "scopes": [
+                "write",
+                "read",
+                "search"
+            ]
+        },
+        "combination": {
+            "scheme": "combination",
```

Comment:
In general, better to name schemes something else like "combo_sc", etc. to avoid confusion.